public/app/features/dashboard/state/DashboardMigrator.test.ts

```diff
@@ -2004,10 +2004,10 @@ describe('DashboardModel', () => {
       model = new DashboardModel({
         templating: {
           list: [
+            // @ts-expect-error
```

Comment:
These `@ts-expect-errors` exists because the migrator expects an older/incomplete version of the variable.